docs/outputs.rst

```diff
@@ -189,19 +189,19 @@ Detections File: Full Output Column Names, Formats, and Descriptions
 +------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
 | phase_deg                          | Float        | The sun-object-observer angle (degrees)                                                                  |
 +------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
-| epochMJD_TDB                       | Float        | Epoch of orbit (MJD) in Barycentric Dynamical Time                                                       |
+| q                                  | Float        | Object perihelion (au)                                                                                   |
 +------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
-| t_p_MJD_TDB                        | Float        | Time of periapsis (MJD)  in Barycentric Dynamical Time                                                   |
+| e                                  | Float        | Orbital eccentricity                                                                                     |
```

Comment:
this depends on what you're doing - I don't have a q because I inputted Keplerian orbit - so I think we need to say orbital elements